cpython/Python/errors.c

```diff
@@ -493,7 +493,11 @@ PyErr_SetFromErrnoWithFilenameObjects(PyObject *exc, PyObject *filenameObject, P
                 MAKELANGID(LANG_NEUTRAL,
                            SUBLANG_DEFAULT),
                            /* Default language */
+#ifndef MS_UWP
```

Comment:
This is needed because we are no longer passing in the FORMAT_MESSAGE_ALLOCATE_BUFFER flag.  When FORMAT_MESSAGE_ALLOCATE_BUFFER  is passed, it expects a wchar**, but when the flag is not specified, it expects a wchar*.  See MSDN for detail.  (Before the change, we actual had a memory corruption bug.)